hermes/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM golang:1.14-alpine as builder
```

```suggestion
FROM golang:1.15-alpine as builder
```

As if it's already building with version 1.15, use the image with the same version